ospd/ospd.py

```diff
@@ -1542,6 +1542,8 @@ def get_vts_xml(self, vt_id=None, filtered_vts=None):
         """
 
         vts_xml = Element('vts')
+        if filtered_vts == -1:
```

Comment:
```suggestion
        if filtered_vts is None:
```

Reply:
Oh just recognized, wouldn't it be possible to just do a 
```suggestion
        if not filtered_vts:
```
here and even drop the get_filtered_vts_list change?